test/__snapshots__/mathml-spec.js.snap

```diff
@@ -204,6 +204,211 @@ exports[`A MathML builder normal spaces render normally 1`] = `
 </math>
 `;
 
+exports[`A MathML builder should build the CD environment properly 1`] = `
+<math xmlns="http://www.w3.org/1998/Math/MathML"
+      display="block"
+>
+  <semantics>
+    <mtable rowspacing="0.24999999999999992em"
+            columnalign="center center center"
+            columnspacing="0.5em"
+    >
+      <mtr>
+        <mtd>
+          <mstyle scriptlevel="0"
+                  displaystyle="true"
+          >
+            <mi>
+              A
+            </mi>
+          </mstyle>
+        </mtd>
+        <mtd>
+          <mstyle scriptlevel="0"
+                  displaystyle="true"
+          >
+            <munderover>
+              <mo stretchy="true"
+                  minsize="3.0em"
+              >
+                →
+              </mo>
+              <mpadded width="+0.6em"
+                       lspace="0.3em"
+              >
+                <mrow>
+                </mrow>
+              </mpadded>
+              <mpadded width="+0.6em"
+                       lspace="0.3em"
+              >
+                <mi>
+                  a
+                </mi>
+              </mpadded>
+            </munderover>
+          </mstyle>
+        </mtd>
+        <mtd>
+          <mstyle scriptlevel="0"
+                  displaystyle="true"
+          >
+            <mi>
+              B
+            </mi>
+          </mstyle>
+        </mtd>
+      </mtr>
+      <mtr>
+        <mtd>
+          <mstyle scriptlevel="0"
+                  displaystyle="true"
+          >
+            <mrow>
+              <mrow>
+                <mstyle displaystyle="false"
+                        scriptlevel="1"
+                >
+                  <mpadded width="0"
+                           lspace="-1width"
```

Comment:
Is `width` supposed to be a number here?

Reply:
It's correct as is. The "-" prefix tells a [lspace](https://developer.mozilla.org/en-US/docs/Web/MathML/Element/mpadded#attr-lspace) attribute that the value is an increment. And `lspace` accepts "1width" as a variable, which is a good thing because we don't know the numeric width.

So this `<mpadded>` creates a right-justified label.